thundra/plugins/log/thundra_log_handler.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+logs = []
+
+
+class ThundraLogHandler(logging.Handler):
+    def __init__(self):
+        logging.Handler.__init__(self)
+        ThundraLogHandler.logs = []
```

Comment:
since `logs` are global variables, they should be cleared here but at log plugin as suggested at previous comments